doc/src/modules/solvers/solveset.rst

```diff
@@ -484,28 +488,28 @@ What will you do with the old solve?
  Hence, it's not yet a perfect replacement for old ``solve``. The ultimate
  goal is to:
 
- * Replace ``solve`` with ``solveset``, by the time solveset is
-   at least powerful as ``solve``, i.e. ``solveset`` does everything
-   that ``solve`` can do currently, and
+ * Replace ``solve`` with ``solveset`` once solveset is at least powerful as
+   ``solve``, i.e., ``solveset`` does everything that ``solve`` can do
+   currently, and
 
- * Eventually rename ``solveset`` to ``solve``.
+ * eventually rename ``solveset`` to ``solve``.
 
 
 How are symbolic parameters handled in solveset?
 ------------------------------------------------
 
- Solveset is in its initial phase of development as of now, so the
- symbolic parameters aren't handled well for all the cases, but some
- work has been done in this regard to depict our ideology towards
- symbolic parameters. As an example the solving of `|x| = n` for `x`
- where `n` is a symbolic parameter. Solveset returns the value of `x`
- considering the domain of the symbolic parameter `n` as well, i.e. :
+ Solveset is in its initial phase of development as of now, so the symbolic
+ parameters aren't handled well for all the cases, but some work has been done
+ in this regard to depict our ideology towards symbolic parameters. As an
+ example, consider the solving of `|x| = n` for real `x`, where `n` is a
+ symbolic parameter. Solveset returns the value of `x` considering the domain
+ of the symbolic parameter `n` as well:
 
- `([0, \infty) ∩ \{n\}) ∪ ((-\infty, 0] ∩ \{-n\})`.
+ .. math:: ([0, \infty) \cap \{n\}) \cup ((-\infty, 0] \cap \{-n\}).
 
- It simply means `n` is the solution only when it belongs to the ``Interval``
- `[0, \infty)` and `-n` is the solution only when `n` belongs to the ``Interval``
- `(- \infty, 0]`.
+ This simply means `n` is the solution only when it belongs to the
+ ``Interval`` `[0, \infty)` and `-n` is the solution only when `-n` belongs to
```

Comment:
n being positive is the same thing as -n being negative, right? So both conditions are saying the same thing and that's going to confuse the alert reader. I would try to pick a different example to use in the docs, especially since this is an Issue.